lua/nvim-treesitter/install.lua

```diff
@@ -259,6 +260,15 @@ local function run_install(cache_folder, install_folder, lang, repo, with_sync,
       )
     end
     return
+  else
+    if not M.ts_generate_args then
+      local ts_cli_version = utils.ts_cli_version()
+      if ts_cli_version and vim.split(ts_cli_version, " ")[1] > "0.20.2" then
```

Comment:
can you make this comparison a helper function and display in healthcheck that the user should update?

Reply:
> LGTM, although it would probably be sufficient to call this function once. It only takes 47ms to invoke the version information so it doesn't matter.

The function is only called once?

Reply:
> can you make this comparison a helper function and display in healthcheck that the user should update?

I deliberately did not, because I did not want to throw away the commit info when building from source. And the point of this check is to be able to work seamlessly even when tree-sitter CLI is not up-to-date, because I did not think we want to bump the _required_ tree-sitter version?